ibis/expr/decompile.py

```diff
@@ -84,6 +83,11 @@ def translate(op, *args, **kwargs):
     raise NotImplementedError(op)
 
 
+@translate.register(ops.TimestampNow)
```

Comment:
is this needed for this PR or just an unrelated fixup? (looks like the right change regardless)

Reply:
It's needed for this PR, but I'm not entirely sure why. I'll poke around a bit.